### PR TITLE
docs: update "About the design system" page

### PR DESCRIPTION
**Pull request by marionnegp**
## What I did

1. Updated all content on the "About the design system" page
2. Deleted "How we build"

## Testing Instructions

1. Check DP

### CODE REVIEW

File: docs/about/index.md [hunk 1]
```diff
@@ -4,179 +4,139 @@ title: About the Design System
...
-  </div>
-  <rh-cta variant="secondary">
-    <a href="/get-started">Get started</a>
-  </rh-cta>
 </section>
+
+## We’re guided by our core principles
+
+<div class="grid sm-two-columns">
+  <rh-card>
```
Comment (marionnegp) [inline docs/about/index.md:41]: Any idea why the icon cards look so funky here? I think there's something going on with the icon in the header slot.

File: docs/about/index.md [hunk 2]
```diff
@@ -4,179 +4,139 @@ title: About the Design System
...
+    <rh-icon slot="header" set="standard" icon="management-and-automation"></rh-icon>
+    <h3 slot="header">Orchestrated</h3>
+    Processes, tools, and roles must be aligned to deliver and maintain an optimal Red Hat digital experience
+  </rh-card>
+</div>
+
+## We rely on our foundations
+
+The Red Hat brand has a strong voice. To make our design system feel like Red Hat, we use foundations like [color](/foundations/color/), [space](foundations/spacing/), and [typography](/foundations/typography/) to enhance our elements and patterns and to align them to Red Hat’s [design language standards](https://www.redhat.com/en/about/brand/standards).
```
Comment (marionnegp) [inline docs/about/index.md:65]: Fix "space" link URL (missing `/` )

File: docs/about/index.md (hunk 1 above)
Comment (bennypowers) [reply docs/about/index.md:48]: probably the aspect-ratio: 1/1

dc127f14 fixes, but these cards should probably have body content

Comment (bennypowers) [reply docs/about/index.md:48]: looks like the cards' body paragraphs weren't marked up properly.

765cb0cd should fix

**Review by bennypowers — APPROVED** [review]

**Pull request merged**